sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/CompatibilityMatrix.java

```diff
@@ -67,6 +70,9 @@ public void generateHtml() throws IOException {
       boolean isLts = center.getSonar().getLtsRelease().equals(sq);
       getSqVersions().add(new SonarVersionModel(sq.getVersion().toString(), displayVersion, releaseDate, isLts));
     }
+
+    getSqVersions().sort(SONAR_VERSION_MODEL_COMPARATOR);
```

Comment:
I don't get why the existing code use `getSqVersions()` to access the List. How about we use the field directly ? :) 
```suggestion
    sqVersions.sort(SONAR_VERSION_MODEL_COMPARATOR);
```